### DISCUSSION TITLE
Is it possible to convert tflite format ?

### DISCUSSION THREAD

**Discussion by cdragon-choi** [snakers4/silero-models#44] [Q&A]
Hello, 
First of all, thanks for great model.
I try to test the model in mobile phone , so i'm trying to convert it tflite format by using tflite converter.
After converting it,  i met some errors like "invalid tflite format" when I load it in mobile application.
Is it possible to convert the tensorflow model to tflite ?

**Comment by snakers4** [comment]
Hi,

We do not really work with TF
Whenever we tried something TF related we always encountered a lot of bugs with cryptic errors, e.g. with tf-js

You can try mobile deployment from pytorch or onnx or try converting from onnx directly 

With tf.js we found out that only reimplementing models from scratch works 

Also note that V2 models are not really mobile ready
